.gitmodules

```diff
@@ -43,3 +43,7 @@
 [submodule "src/popscle"]
 	path = src/popscle
 	url = https://github.com/vib-singlecell-nf/popscle.git
+[submodule "src/scrublet"]
+	path = src/scrublet
+	url = git@github.com:vib-singlecell-nf/scrublet.git
```

Comment:
For consistency we should move all of these to ssh based rather than https based